packages/loader/src/ShaderChunkLoader.ts

```
@@ -0,0 +1,45 @@
+    const { url } = item;
+
+    return this.request<string>(url, resourceManager, { ...item, type: "text" }).then((code) => {
+      ShaderFactory.registerInclude(url.substring(1), code);
+
+      return _loadChunksInCode(code, url, resourceManager);
+    });
+  }
+}
+
+/** @internal */
+export function _loadChunksInCode(code: string, basePath: string, resourceManager: ResourceManager): Promise<void[]> {
```

Biome

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)